packages/react/src/components/DashboardEditor/DashboardEditorHeader/DashboardEditorHeader.jsx

```diff
@@ -220,13 +234,61 @@ const DashboardEditorHeader = ({
     </div>
   );
 
+  // handle the edit title button
+  const handleEditClick = useCallback(() => {
+    setIsTitleEditMode(true);
+  }, []);
+
+  const editWidgets = useMemo(
```

Comment:
nit: I don't think the input and buttons are considered widgets

Reply:
thanks, will rename

Reply:
thanks!